index.html

```diff
@@ -120,7 +124,17 @@
         *   Make sure to not add '' around this that aren't strings.
         */
         if (isString(formattedOut)) {
-          term.echo(`'${formattedOut}'`);
+          if (formattedOut.indexOf('\n') == -1) {
+            /*
+            * output includes no newline characters
+            */
+            term.echo(`'${formattedOut}'`);
+          } else {
+            /*
+            * output includes newline characters
+            */
+            term.echo(`"""${formattedOut}"""`)
```

Comment:
I can't seem to get this to fire. What input have you given it where this works?

Reply:
It seems like this block was never executed, therefore it never worked. But, I moved it around to another block and now it works!